### DISCUSSION TITLE
Ideas and features

### DISCUSSION THREAD

**Discussion by abh80** [abh80/smart-edge#15] [Ideas]
# Ideas and features
Share any ideas and features you have in mind 💡here, to make the experience better 🙂.

**Comment by Jo754709** [comment]
Hello, the app looks really good. The following features would be a great addition to the app;
1. Ability to slide between all notifications when the smart edge is expanded.
2. Show notification for incoming calls showing the callers name, the accept and reject button.
3. Show visualiser when recording audio.
4. Support for different punch hole orientations (the left, right, and 'middle)
5. The ability to adjust the view height.
6. The ability to choose or select what apps should the smart wedge show.
7. The ability to show animation when device connected to charger and also for bluetooth headsets.
8. The ability to surround the punch hole camera when there are no any notifications.

Also, there is a bug I noticed. Any app icon (especially the default messaging app) on the right is not clickable at all. Only app icons on the left are clickable.

**Reply by abh80** [reply]
Thank you for your suggestions I will definitely work on them, apart from that the bug will also be fixed the notification is buggy at the moment i am working on that

**Reply by PeoPessi** [reply]
I have added `Ability to slide between all notifications when the smart edge is expanded.` in the latest alpha9 release

**Reply by Jo754709** [reply]
That is awesome! Thank you

**Comment by dghqz** [comment]
Hi! I've been using the app and so far it has been great, however it would be great of you can add a couple of features:

- The capability of whitelist/blacklist individual apps
- Light and dark mode switch for the app, not the dynamic island (Light mode might come handy when editing the overlay layout)
- Show/scroll the title of the song playing (and a toggle switch to enable or disable it)
- Normal and minimal layout presets (For a just icons or detailed information view)
- Border lightning inside the dynamic island based on the app icon when a new notification shows up (Not extremely necessary but it would be a neat addition)

Also I noticed a small visual bug when you play a YouTube video, instead of showing the YT logo it shows the thumbnail of the video playing, however it's not a big deal and nothing that interferes with its functionality.

**Comment by abh80** [comment]
Thanks for all the support managed to reach 100+ stars 🎉 🥳 , Love you all ❤️

**Reply by Jo754709** [reply]
You are welcome

**Comment by unilock** [comment]
I just found this app while idly browsing F-Droid (specficially IzzyOnDroid's repo). It's so cool!!
The only unique suggestion I have right now would be: the ability to close the "expanded" island (while it's showing media controls) by tapping *inside* of it, rather than / including *below* it (as in, lower on the screen).
(and I'd like to second the "adjustable island height" idea as well :)
...which I assume would require resizing the icons on the island - maybe this could be added as a separate option, too?)

**Reply by unilock** [reply]
It would be nice to be able to hide the island completely while it's not showing anything, too :)

**Reply by abh80** [reply]
You can set minimum width to zero under overlay manage settings

**Reply by unilock** [reply]
Ah, I'm blind and missed the "Height of overlay when shrinked" and "Space between icons when overlay is shrinked" options. (I had increased "Minimum Width of overlay when closed" because the icons were partially under the camera while the overlay was shrunken, but increasing the "space between icons" works for that too)
Thanks! :)
